### PR TITLE
layers: Add name & getters to LCM

### PR DESCRIPTION
**Pull request by gustafj**
Adds getter for fingerprint of a LCM layer, getter for all Supported
LCM fingerprints and the string representation of the LCM name when
registering a new LCM payload layer.

Co-Authored-By: TacoVox <jonas@derkahler.de>

### CODE REVIEW

**Comment by TacoVox** [comment]
I am obviously fine with this P/R.
I give my consent.

**Comment by TacoVox** [comment]
@gconnell small poke :sunglasses:

File: layers/lcm.go [hunk 1]
```diff
@@ -191,3 +206,8 @@ func (lcm LCM) LayerPayload() []byte {
 func (lcm LCM) Payload() []byte {
 	return lcm.LayerPayload()
 }
+
+// Fingerprint returns the LCM fingerprint of the underlying message.
+func (lcm LCM) Fingerprint() LCMFingerprint {
+	return lcm.fingerprint
```
Comment (gconnell) [inline layers/lcm.go]: just export (capitalize) this value?  that way folks can set it as well as get it, which would be nice if we ever make this serializable.

Comment (gustafj) [reply layers/lcm.go]: @gconnell thanks for the response! :sunglasses: 
That would be a larger change, and have effect on the `lcmLayerTypes` map. I agree that serializeable is nice, but that is IMHO out of scope for this change. 
I'd rather just have it as is in this PR, and if / when its made serializeable we can reevaluate.

**Comment by gconnell** [comment]
Sounds good

**Pull request merged**